projects/iot-cloud/docker/stack.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+services:
+    broker:
+        image: beia/mqtt-broker
+        volumes:
+         - mosca_data:/db
+        environment:
+         - REDIS_HOST=redis
+        depends_on:
+         - redis
+        ports:
+         - "1883:1883"
+         - "8080:80"
```

Comment:
Ar trebui sa fie rulat prin docker proxy. Se va inbunatati mai tarziu

Reply:
si securizat https://www.techrepublic.com/article/75-of-public-facing-redis-servers-are-infected-with-malware-heres-how-to-fix-it/

Reply:
@georgesuciu Suntem ok cu securitatea Redis. Porturile lui nu sunt expuse public, ci poate comunica doar cu celelalte containere din stack. 

Ce putem face în plus, este să folosim o parolă, respectiv să izolăm containerele în împărțim containerele în mai multe rețele, dar niciuna din aceste chestii nu sunt vitale. Sunt doar hardenings în cazul în care deja un serviciu (care nu folosește redis) este compromis.